src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -159,85 +153,89 @@ google.auth.getApplicationDefault(function(err, authClient) {
 
   logging.monitoredResourceDescriptors.list(request, handlePage);
 });
+
+function authorize(callback) {
+  new GoogleAuth().getApplicationDefault(function(err, authClient) {
+    if (err) {
+      console.log('authentication failed: ', err);
+      return;
+    }
+    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
+      var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
+      authClient = authClient.createScoped(scopes);
+    }
+    callback(authClient);
+  });
+}
 // BEFORE RUNNING:
 // ---------------
 // 1. If not already done, enable the Google Cloud Logging API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/logging
-// 2. This sample uses Application Default Credentials for authentication.
+// 2. Install the Node.js client library by running
+//    `npm install googleapis --save`
+// 3. This sample uses Application Default Credentials for authentication.
 //    If not already done, install the gcloud CLI from
-//    https://cloud.google.com/sdk/ and run
+//    https://cloud.google.com/sdk and run
 //    `gcloud beta auth application-default login`
-// 3. Install the Node.js client library and Application Default Credentials
-//    library by running `npm install googleapis --save`
+
 var google = require('googleapis');
-var logging = google.logging('v2beta1');
+var GoogleAuth = require('google-auth-library');
 
-// Use Application Default Credentials for authentication when running locally.
-// For more information, see:
-// https://developers.google.com/identity/protocols/application-default-credentials
-google.auth.getApplicationDefault(function(err, authClient) {
-  if (err) {
-    console.log(err);
-    return;
-  }
-  if (authClient.createScopedRequired && authClient.createScopedRequired()) {
-    var scopes = ['https://www.googleapis.com/auth/cloud-platform'];
-    authClient = authClient.createScoped(scopes);
-  }
+var logging = google.logging('v2beta1');
 
+authorize(function(authClient) {
   var request = {
-
-    // TODO: Change placeholder below to desired parameter value for the `delete` method:
-
     // Required. The resource name of the log to delete. Example: `"projects/my-project/logs/syslog"`.
-    logName: 'projects/{MY-PROJECT}/logs/{MY-LOG}',
-
+    logName: '', // TODO: Update placeholder value(s).
```

Comment:
Per offline discussion with @saicheems, suggested implementing the latter for now, with an issue filed to explore the former.